iwyu_output.h

```diff
@@ -235,10 +235,14 @@ class IwyuFileInfo {
                            const string& symbol);
   // TODO(dsturtevant): Can we determine in_cxx_method_body? Do we care?
 
+  // Called when using a macro in this file.
   void ReportMacroUse(clang::SourceLocation use_loc,
                       clang::SourceLocation dfn_loc,
                       const string& symbol);
 
+  // Called when somebody uses a macro defined in this file.
+  void ReportDefinedMacroUse(const clang::FileEntry* used_in);
```

Comment:
It bugs me a little that you can derive `used_in` already from the arguments to `ReportMacroUse`, but I see that you only call `ReportMacroUse` if `ShouldReportIWYUViolationsFor` is true, so it's hard to unify this.